firmware/boron-ins-fsm/test/mocks/helper.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Contains port of Linux function clock_gettime(CLOCK_MONOTONIC)
+ * To be used in millis() mock function for Windows machines
+ */
```

Comment:
Cool. Was this copied from somewhere? If so, it would be good to add the link in the comments.

Reply:
Yep, I added a link to the source.